wasm/lib/README.md

```diff
@@ -7,6 +7,10 @@ A Python-3 (CPython >= 3.5.0) Interpreter written in Rust.
 [![Contributors](https://img.shields.io/github/contributors/RustPython/RustPython.svg)](https://github.com/RustPython/RustPython/graphs/contributors)
 [![Gitter](https://badges.gitter.im/RustPython/Lobby.svg)](https://gitter.im/rustpython/Lobby)
 
+# WARNING: this project is still in an pre-alpha state!
```

Comment:
```suggestion
# WARNING: this project is still in a pre-alpha state!
```

Reply:
Oh whoops, I originally had it say "an alpha state". Thanks!